swift/swift.py

```diff
@@ -3,7 +3,7 @@
 """
 Swift is a main manager for swift system.
 
-Using a set-up file created by an user, it sets up apps and buses.
+Using a set-up file written by an user, it sets up apps and buses.

```

Comment:
Oh isn't it "a user" rather than "an user"? 😄 I noticed that now...

Reply:
Wow, "a user" is the right expression!